zerver/views/webhooks/greenhouse.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from django.utils.translation import ugettext as _
+from django.http import HttpRequest, HttpResponse
+from typing import Any
+
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.models import UserProfile, Client
+
+import ujson
+
+MESSAGE_TEMPLATE = "Applying for role:\n{}\n**Emails:**\n{}\n\n>**Attachments:**\n{}"
+
+def list_format(some_list):
+    # type: (List[Any]) -> str
+    internal_template = ''
+    for item in some_list:
+        item_type = item.get('type').title()
```

Comment:
its a little bit risky.
maybe could we do it like:
`item.get('type', '').title()`?